.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
That file seems unrelated to the problem. Let's not have this change in this PR.